vtStorAtaManaged/AtaCommandExtensionsManaged.cpp

```diff
@@ -113,6 +113,15 @@ namespace vtStor
                 errorCode = static_cast<vtStor::Managed::eErrorCode>(m_AtaCommandExtensions->IssueCommand_DownloadMicrocodeDma(spDriveInterface, CommandType, spBuffferInterface, SubCommand, BlockCount, BufferOffset));
                 return(errorCode);
             }
+
+            vtStor::Managed::eErrorCode cAtaCommandExtensions::IssueCommand_ATATrim(vtStor::Managed::cDriveInterface^ Drive, U32 CommandType, vtStor::Managed::cBufferInterface^ Data, std::vector<sLBARangeEntry> LBARangeEntries)
```

Comment:
i will define a object have a LBARangeEntries attribute (std::vector<sLBARangeEntry> LBARangeEntries) to use for parameter